var/spack/repos/builtin/packages/netgen/package.py

```diff
@@ -0,0 +1,74 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Netgen(AutotoolsPackage):
+    """NETGEN is an automatic 3d tetrahedral mesh generator. It accepts
+       input from constructive solid geometry (CSG) or boundary
+       representation (BRep) from STL file format. The connection to
+       a geometry kernel allows the handling of IGES and STEP files.
+       NETGEN contains modules for mesh optimization and hierarchical
+       mesh refinement. """
+
+    # FIXME: Add a proper url for your package's homepage here.
+    homepage = "https://ngsolve.org/"
+    url = "https://gigenet.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.3/netgen-5.3.1.tar.gz"
+
+    version('5.3.1', 'afd5a9b0b1296c242a9c554f06af6510')
+
+    variant("mpi", default=True, description='enable mpi support')
+    variant("oce", default=False, description='enable oce geometry kernel')
+    variant("nogui", default=True, description='disable gui')
```

Comment:
It's probably more logical to call the variant `gui`, so that `+gui` enables the GUI and `~gui` disables the GUI.